testing/run-unit-tests-on-cluster.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+CURRENT_CONTEXT=$(kubectl config current-context)
```

Comment:
1) with this script setting labels always means running on external, what to do to set labels and run on default env-test? ;) Let's just not run tests in this script but just set environment variables.

2) Then do not do any cleanups etc on that cluster. Just mention in the documentation, that the cluster should be clean and the user needs to cover that.

Reply:
1. Running on envtest cluster CLI could be done by `make test` (covered in doc)
2. Done
3. Done
4. Done, my only reservation is that in this case we create "intermediary" script setting envs, that is always called (from IDE, from GA, and from CLI). This is potential risk that if someone commit the overidden env settings from local version of script, it could break GA. E.g. if setting existing cluster got committed and GA would try to execute unit tests on existing cluster.

Reply:
Changes pushed.